### PR TITLE
Use constants for common strings

### PR DESCRIPTION
**Pull request by mike182uk**
In https://github.com/TryGhost/ActivityPub/pull/5 we introduced constants to keep track of common strings. This PR updates the codebase to make use of these constants

### CODE REVIEW

File: src/user.ts [hunk 1]
```diff
@@ -61,18 +66,6 @@ export async function getUserData(ctx: RequestContext<ContextData>, handle: stri
         ),
     };
 
-    const dataToStore: PersonData = {
```
Comment (mike182uk) [inline src/user.ts:64]: Removed this as it wasn't being used

Comment (allouis) [reply src/user.ts:64]: :D This is actually a bug - we should be passing `dataToStore` on line 76 🙈

**Review by allouis — COMMENTED** [review]
Looks good - but we wanna keep `dataToStore` (and actually use it!)

File: src/user.ts (hunk 1 above)
Comment (mike182uk) [reply src/user.ts]: Updated in https://github.com/TryGhost/ActivityPub/pull/6/commits/a32d9356488ea158df674275a553982ab60ea2ff

**Review by allouis — APPROVED** [review]

**Pull request merged**